for-organizations.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
+  <head>
+    <meta charset="utf-8" />
+    <title>For Organizations - Solid</title>
+    <meta content="width=device-width, initial-scale=1" name="viewport" />
+    <link rel="stylesheet" href="style/base.css" />
+  </head>
+
+  <body about="" prefix="schema: http://schema.org/ doap: http://usefulinc.com/ns/doap# foaf: http://xmlns.com/foaf/0.1/" id="for-organizations">
+    <header>
+      <address>
+        <a href="/"><img alt="Solid logo" src="/image/logo.svg" width="50" /> Solid</a>
+      </address>
+
+      <nav>
+        <ul>
+          <li><a href="/about.html">About Solid</a></li>
+          <li><a href="/for-developers.html">For Developers</a></li>
+          <li><a href="/for-organizations.html">For Organizations</a></li>
+          <li><a href="/community.html">Community</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <article about="" typeof="schema:Article">
+        <h1 property="schema:name">Organizations using Solid</h1>
+
+        <p>These organizations work in a range of capacities to support the deployment and research
+                of Solid technology.</p>
+
+        <div datatype="rdf:HTML" property="schema:description">
+          <section id="solid-organizations" inlist="" rel="schema:hasPart" resource="#solid-organizations">
+            <h2 property="schema:name">Solid Organizations</h2>
+            <div datatype="rdf:HTML" property="schema:description">
+              <dl class="legend">
+                <dt><img src="image/hexagon.png" alt="hexagon" width="20" height="20" /></dt>
+                <dd>Pod provider</dd>
+                <dt><img src="image/research.png" alt="research" width="25" height="25" /></dt>
+                <dd>Research institutions</dd>
+              </dl>
+
+              <ul>
+                <li class="pod-provider">
+                  <h3>BBC Research &amp; Development</h3>
+
+                  <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+
+                  <details>
```

Comment:
I'm curious about this. It's not part of the design. Is it needed for RDF or accessibility? I think we will want to display the content in a different way (without the labels/accordion), is that possible?